detectors/aws/ecs/ecs_test.go

```diff
@@ -100,6 +207,13 @@ func TestDetectCannotReadContainerName(t *testing.T) {
 	assert.Equal(t, 0, len(res.Attributes()))
 }
 
+func TestDetectV4Parse(t *testing.T) {
```

Comment:
What is the purpose of this test? It makes no assertions.

Reply:
Yeah it was spurious, removed with 3490115